ts/Series/ErrorBar/ErrorBarPoint.d.ts

```diff
@@ -0,0 +1,42 @@
+/* *
+ *
+ *  (c) 2010-2020 Torstein Honsi
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type BoxPlotPoint from '../BoxPlot/BoxPlotPoint';
+import type BoxPlotPointOptions from '../BoxPlot/BoxPlotPointOptions';
+import type ErrorBarSeries from '../ErrorBar/ErrorBarSeries';
+
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+declare class ErrorBarPoint extends BoxPlotPoint {
+    public options: ErrorBarPointOptions;
+    public series: ErrorBarSeries;
+}
+
+export interface ErrorBarPointOptions extends BoxPlotPointOptions {
+}
+
```

Comment:
Please put the point options in a separate definition file. ErrorBarPoint should only be about the class, so that we can refactor better later on, when we need a real point class.